site/content/docs/browser-checks/_index.md

```diff
@@ -258,6 +258,15 @@ test('Open the page and take a screenshot', async ({ page }) => {
 {{< /tab >}}
 {{< /tabs >}}
 
+### Static Analysis Requirement
+{{< info >}}
```

Comment:
Can we simplify it at all? I think we can leave out "Static Analysis" for example. I think 99% of users won't bump into this, so I think it's ok to leave out some detail to make it simpler. 

> Checkly inspects your code to detect which browser is used. To use Chrome, please set `channel: 'chrome'` without any intermediate variables or functions. For example, using a variable like `test.use({ channel: chromeChannel })` can cause Checkly to fail to detect that Chrome is being used.